### PR TITLE
A few more Integer Functions

### PR DESCRIPTION
**Pull request by rocky**
* CompositeQ
* Divisor
* ModularInverse

Also:
* move BernoulliB to a more suitable place in the code
* some attribute A_'s added

<a href="https://gitpod.io/#https://github.com/Mathics3/mathics-core/pull/427"></a>

### CODE REVIEW

**Comment by rocky** [comment]
Merging in and continuing since there are many other things to do. However we'll keep the branch open, for review.

Also, function properties may need adjusting.

File: mathics/builtin/intfns/divlike.py [hunk 1]
```diff
@@ -192,6 +241,43 @@ def apply(self, n: Integer, m: Integer, evaluation):
...
 
 
+class ModularInverse(SympyFunction):
+    """
+    <dl>
+      <dt>'ModularInverse[$k$, $n$]'
+      <dd>returns $k$ modulo $n$.
+    </dl>
+
+    'ModularInverse[$k$,$n$]' gives the number $r$ such that the remainder of the division of $r$ x $k$ by $n$ is equal to 1.
```
Comment (TiagoCavalcante) [inline mathics/builtin/intfns/divlike.py:251]: Maybe "gives the smallest number"?

Comment (rocky) [reply mathics/builtin/intfns/divlike.py:251]: Sure. See c7b0faba

File: mathics/builtin/intfns/divlike.py [hunk 2]
```diff
@@ -192,6 +241,43 @@ def apply(self, n: Integer, m: Integer, evaluation):
         return Integer(n % m)
 
 
+class ModularInverse(SympyFunction):
+    """
+    <dl>
+      <dt>'ModularInverse[$k$, $n$]'
+      <dd>returns $k$ modulo $n$.
```
Comment (axkr) [inline mathics/builtin/intfns/divlike.py:248]: returns the modular inverse k^(-1) mod n.

Comment (rocky) [reply mathics/builtin/intfns/divlike.py:248]: Ok - good. See 105803c8

**Pull request merged**